extensions-core/kinesis-indexing-service/src/main/java/org/apache/druid/indexing/kinesis/supervisor/KinesisSupervisor.java

```diff
@@ -378,6 +379,13 @@ protected boolean useExclusiveStartSequenceNumberForNonFirstSequence()
     return true;
   }
 
+  // not yet supported, will be implemented in the future maybe? need to find a proper way to measure kinesis lag.
+  @Override
+  public LagStats computeLagStats()
```

Comment:
Not sure if UnsupportedOperationException would be better here or if null is fine.

Reply:
Sure && Done.